types/context.go

```diff
@@ -141,6 +142,8 @@ const (
 	contextKeySigningValidators
 	contextKeyGasMeter
 	contextKeyMinimumFees
+	contextKeyKVGasConfig
+	contextKeyTransientGasConfig
```

Comment:
Do we need to store the Transient Gas Config separately? I think we should remove this for now (for code clarity), until we plan on having on the Transient Store's reads cost a different amount than normal stores. AFAIK, this is a very post launch consideration.